config/v1/types_infrastructure.go

```diff
@@ -67,7 +67,7 @@ type InfrastructureStatus struct {
 	EtcdDiscoveryDomain string `json:"etcdDiscoveryDomain"`
 
 	// apiServerURL is a valid URI with scheme(http/https), address and
-	// port.  apiServerURL can be used by components like the web console
+	// port(only if not using default port HTTP 80 and HTTP/TLS 443).  apiServerURL can be used by components like the web console
```

Comment:
How about `... and optionally a port (defaulting to 80 for http and 443 for https)`?

You also have to generate the CRDs. `make update` or something like that. `make help` shows you the commands.

Reply:
@sttts  Thanks for your suggestion.
I have updated the api doc and updated CRD by calling `make update-codegen-crds`